app/models/ccms/submission.rb

```diff
@@ -20,14 +20,14 @@ def process!(options = {}) # rubocop:disable Metrics/MethodLength
         ObtainApplicantReferenceService.call(self)
       when 'applicant_submitted'
         CheckApplicantStatusService.call(self)
-      when 'applicant_ref_obtained'
```

Comment:
Any reason why these lines were moved? I don't think it's a problem, just curious.

Reply:
The change to the order more accurately covers the order in which the requests are made